src/prelude/prim.mo

```diff
@@ -824,6 +824,24 @@ func getCandidLimits<system>() : {
   };
 };
 
+func setCandidTypeLimits<system>({
```

Comment:
exposing this method to the user seems like a bit too much power/opportunity to do wrong things; no strong opinions but I wonder if this shouldn't perhaps be more like a compile-time setting, perhaps?